var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -362,11 +362,10 @@ def patch_hdf5_pkgconfigcmake(self):
             return
 
         pkgconfig_file = find(self.prefix, "netcdf.pc", recursive=True)
-        cmakeconfig_file = find(self.prefix, "netCDFTargets.cmake", recursive=True)
```

Comment:
The comment on line 357 should have "and config.cmake" removed now the cmake part is no longer done